app/components-react/pages/onboarding/Connect.tsx

```diff
@@ -47,88 +63,176 @@ export function Connect() {
-          </p>
-        )}
-        {isRelog && (
-          <h3 style={{ marginBottom: '16px' }}>
-            {$t(
-              'Your login has expired. Please reauthenticate to continue using Streamlabs Desktop.',
+        <h1 className={commonStyles.titleContainer} style={{ marginTop: 0 }}>
+          {title}
+        </h1>
+        {isSignup ? (
+          <Signup onSignupLinkClick={() => setIsSignup(false)} />
+        ) : (
+          <>
+            {!isRelog && <p style={{ marginBottom: 20 }}>{$t('Log in with email/password')}</p>}
+            {isRelog && (
+              <h3 style={{ marginBottom: '16px' }}>
+                {$t(
+                  'Your login has expired. Please reauthenticate to continue using Streamlabs Desktop.',
+                )}
+              </h3>
             )}
-          </h3>
+            <div className={styles.streamlabsPlatformContainer}>
+              <PlatformButton
+                platform="streamlabs"
+                loading={loading}
+                disabled={loading || authInProgress}
+                onClick={() => authPlatform('streamlabs', afterLogin)}
+              >
+                <Translate
+                  message={$t('Log in with <span>%{platform}</span>', {
+                    platform: 'Streamlabs ID',
+                  })}
+                >
+                  <span slot="span" style={{ fontWeight: 'bold' }} />
+                </Translate>
+              </PlatformButton>
+            </div>
+
+            <div className={styles.thirdPartyPlatforms}>
+              <p>{$t('Log in with a platform')}</p>
+
+              <div className={styles.thirdPartyPlatformsContainer}>
+                {platforms.map((platform: TPlatform) => (
+                  <PlatformButton
+                    platform={platform}
+                    disabled={loading || authInProgress}
+                    loading={loading}
+                    onClick={() => authPlatform(platform, afterLogin)}
+                    key={platform}
+                  >
+                    <Translate
+                      message={$t('Log in with <span>%{platform}</span>', {
+                        platform: platformLabels(platform),
+                      })}
+                    >
+                      <span slot="span" />
+                    </Translate>
+                  </PlatformButton>
+                ))}
+              </div>
+              <div className={styles.extraPlatformsContainer}>
+                {shouldAddTrovo && (
+                  <PlatformIconButton
+                    platform={EPlatform.Trovo}
+                    disabled={loading || authInProgress}
+                    loading={loading}
+                    onClick={() => authPlatform(EPlatform.Trovo, afterLogin)}
+                    title={$t('Log in with %{platform}', {
+                      platform: platformLabels(EPlatform.Trovo),
+                    })}
+                  />
+                )}
```

Comment:
i feel like we could maybe just add trovo to the `extraPlatforms` array if the condition is met

Reply:
I don't remember exactly but I think that required more changes (like in the Extra Platforms controller) since this one probably connect as opposed to the other ones that just set the Stream settings.